chains/solana/contracts/programs/example-lockrelease-token-pool/src/lib.rs

```diff
@@ -0,0 +1,314 @@
+use anchor_lang::prelude::*;
+use anchor_lang::solana_program::program::invoke_signed;
+use anchor_spl::token_2022::spl_token_2022::{self, instruction::transfer_checked};
+use example_burnmint_token_pool::{common::*, rate_limiter::*};
```

Comment:
It's a little bit odd for the user to copy those from the other example, if the common files are in a separate module, then the user will need to copy the two folders and it will be already working. Wdyt?